src/System.Runtime.InteropServices/ref/System.Runtime.InteropServices.cs

```diff
@@ -575,6 +579,8 @@ public static partial class Marshal
         [System.Security.SecurityCriticalAttribute]
         public static System.IntPtr StringToCoTaskMemUni(string s) { return default(System.IntPtr); }
         [System.Security.SecurityCriticalAttribute]
+        public static System.IntPtr StringToCoTaskMemUTF8(string s) { return default(System.IntPtr); }
```

Comment:
Also update the version in the CSProj and change the NETStandard version of this project.